include/nrm/utils/scopes.h

```diff
@@ -30,7 +30,10 @@ nrm_scope_t *nrm_scope_create(void);
 #define NRM_SCOPE_TYPE_MAX 3
 
 /**
- * Add an int to the list
+ * Add an int corresponding to some hardware ID to the list
+ * @param *:
```

Comment:
name the param in that case (change the prototype of the function) , because that doesn't work in doxygen.